### DISCUSSION TITLE
光猫IP和OP的IP冲突-L大解决方法

### DISCUSSION THREAD

**Discussion by loopvoid** [coolsnowwolf/lede#9512] [R68S / 电犀牛专区]
# 光猫IP和OP的IP冲突
1. 创建一个 MAN 的接口
![QQ图片20220611144818](https://user-images.githubusercontent.com/15059763/173177164-a1241c5b-4555-4dba-96fd-0c66fdb4a7d7.png)
2. 绑定 ETH1 ，就是 WAN
然后静态地址，同一段，不要给网关和 DNS
![QQ图片20220611144853](https://user-images.githubusercontent.com/15059763/173177173-acfb564d-4feb-4e60-a958-6027c4e1aee9.png)
3. 防火墙设置到 WAN 的 zone，保存。
![QQ图片20220611144932](https://user-images.githubusercontent.com/15059763/173177179-ef46a501-5217-4571-b93e-46913f180548.png)

**Comment by github-h** [comment]
补充一下，上面的教程有点小问题，大雕可能写错了

**IPv4那里应该设置为光猫的同一网段 例如 192.168.1.2**

原理是创建了一个跟光猫同一个网段的wan口，所以当然ip应该是光猫的那个网段

不应该是lan口下的100段

其他步骤均相同
![图片](https://user-images.githubusercontent.com/49114731/173239095-b99cdbda-06e6-40ff-b229-002037e4cb25.png)
